tox.ini

```diff
@@ -84,8 +89,24 @@ commands =
 
 [testenv:flake8]
 basepython = python2.7
-deps = flake8
-commands = flake8 --doctests src tests setup.py
+deps =
+    flake8
+    flake8-html
+    flake8-coding
+    flake8-debugger
+    flake8-deprecated
+    flake8-isort
+    flake8-pep3101
+    flake8-plone-hasattr
+    flake8-polyfill
+    flake8-print
+    flake8-quotes
+    flake8-string-format
+    flake8-todo
+
+commands =
+    - flake8 --format=html --htmldir={toxinidir}/reports/flake8 --doctests src tests setup.py {posargs}
```

Comment:
the "-" indactaes tox that it should ignore the exit code for passing or failing of the test. 
see https://tox.readthedocs.io/en/latest/example/basic.html#ignoring-a-command-exit-code 
as flake8 invoked with --format=html and -htmldir it did not print anything on the commandline, as the test has already failed, when there is a violation. so that double is just to generate both outputs.